### DISCUSSION TITLE
Support for XG-Flash (GBA)?

### DISCUSSION THREAD

**Discussion by Akamaru** [sanni/cartreader#130] [Ideas & Suggestions]
Did the cartreader support the XG-Flash for GBA?
I recently got one without the Rom Linker, so I was wondering if the cartreader will support it?
I dont know how the XG-Flash work but if I can help you can ask me to try something.
Some info about the XG-Flash with Downloads: https://www.gameboy-advance.net/flash_linker/gba_xg-flash.htm https://www.emu-zone.net/gba-sp/xg-flash.htm

**Comment by sanni** [comment]
It does not support it right now but maybe someone adds it in the future. It shouldn't be too hard since there are already a few different flash chips supported in the code and the chips used in the XG-Flash might be similar.
